docs/modules/ROOT/pages/support.adoc

```diff
@@ -1,33 +1,38 @@
 = Support
-:url-slack-hazelcast: https://join.slack.com/t/hazelcastcommunity/shared_invite/zt-feevu2bl-AWtL_N~pc7fGvdO9iKJAww
 :url-zendesk-hazelcast: https://hazelcast.zendesk.com/
 
 Hazelcast Cloud has different support options depending on your requirements. Here are the support levels:
 
-== Community
-
-Community support is available for free tier users. 
-*Support Channel:* link:{url-slack-hazelcast}[Slack]
-
 == Standard
 
 Standard support is available for all users with non-free clusters.
+
 *Coverage:* 24 x 7 x 365
+
 *Support Channel:* link:{url-zendesk-hazelcast}[ZenDesk]
-*SLA:* First Response < 1 Business Day
+
+*SLA:*
+
+First Response < 1 Business Day
 
 == Enterprise
 
-For enterprise support please email to mailto:sales@hazelcast.com[]. 
+For enterprise support please email to mailto:sales@hazelcast.com[].
+
 *Coverage:* 24 x 7 x 365
-*Support channels: * link:{url-zendesk-hazelcast}[ZenDesk], Emergency Telephone Number
-*Emergency Patches*
+
+*Support channels:* link:{url-zendesk-hazelcast}[ZenDesk], Emergency Telephone Number
+
 *SLA*
+
 Severity 1: First Response < 30mins (24 x 7 x 365)
```

Comment:
Perhaps we could give users an indication of what we mean by these severity levels?

Also why do we repeat '(24 x 7 x 365)'? We have already mentioned that under 'Coverage'.

Reply:
Could you please check the indications added?

Reply:
Looks good, thanks. I just added some more context because high, medium, and low impact is ambiguous.

Reply:
All accepted, thanks a lot @JakeSCahill !!!